backend/sh/copy_file_to_image.sh

```diff
@@ -59,13 +59,14 @@ if [ "${p}" != "boot" ]; then
     e2mkdir /tmp/${uuid}/${p}.img:$(dirname ${o})
     e2cp ${i} /tmp/${uuid}/${p}.img:${o}
 else
-    if [ ! -d $(dirname ${o}) ]; then
-        d_echo "mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o})"
-        mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o})
-    fi
+    # ToDo: mmd only supports adding one single directory to a existing one.
+    # consequently the current code would fail when adding a directory with sub dir(s).
+    # calling mmd recursively for each dir would be a solution. 
+    d_echo "mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o})"
+    mmd -D sS -i /tmp/"${uuid}"/"${p}".img ::$(dirname "${o}") || d_echo "mmd failed: maybe the directory already exists?"
```

Comment:
what does this improve? you can not check if a file exists with the omnect-cli interface. so the user still doesnt know if the file copy worked.

Reply:
what happens if the file already exists? does mcopy overwrite?
why leave the "ToDo"?

Reply:
> what happens if the file already exists? does mcopy overwrite? 

yes, mcopy overwrites.

> why leave the "ToDo"?

it is effort and is it really needed for boot partition at the moment? we would see an error in case. are we sure it works on other partitons with `e2mkdir`? -> I'll check this...

Reply:
`e2mkdir`works

Reply:
fixed ToDo